Tests/PostgresNIOTests/New/Extensions/LoggingUtils.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 07.01.21.
+//
+
+import Logging
```

Comment:
Ditch Xcode template headers.
```suggestion
import Logging
```